drivers/volume/common.go

```diff
@@ -658,6 +658,14 @@ func (d *DefaultDriver) GetAutoFsTrimStatus(pxEndpoint string) (map[string]api.F
 	}
 }
 
+// GetAutoFsTrimUsage get autofstrim  usage stats
+func (d *DefaultDriver) GetAutoFsTrimUsage(pxEndpoint string) (map[string]*api.FstrimVolumeUsageInfo, error) {
+	return nil, &errors.ErrNotSupported{
```

Comment:
it is a interface method, will be called for drivers where this not implemented. since is error "not supported" is thrown, I don't think there is a need to created empty map